### DISCUSSION TITLE
Is spotiflyer available for windows 10 ?

### DISCUSSION THREAD

**Discussion by strider-ape** [Shabinder/SpotiFlyer#853] [General]
I am a spotiflyer user on android.
I want to download my whole Spotify playlist on my pc with windows 10

**Comment by ElizabethWega** [comment]
> I am a spotiflyer user on android.
> I want to download my whole Spotify playlist on my pc with windows 10

![IMG_20220114_202710.jpg](https://user-images.githubusercontent.com/88385659/149573784-4b7c83f7-6aa6-42c6-9905-965b8105633c.jpg)
The answer is yes. Go to the main page of SpotiFlyer app on GitHub. Scroll down, to the section called 'Install'. Press the button near the 'Windows' edition and it will immediately download the file on your computer.

**Reply by strider-ape** [reply]
thank you for giving info
